app/src/shared/components/SubscriptionStatusContextProvider/Subscription.jsx

```diff
@@ -18,7 +21,36 @@ const Message = {
     Warning: 'MW',
 }
 
-class Subscription extends Component {
+type SubscriptionStatus = {
+    register: Function,
+    unregister: Function,
+    subscribed: Function,
+    unsubscribed: Function,
+}
+
+type Props = {
+    onMessage: Function,
+    onDoneMessage: Function,
+    onErrorMessage: Function,
+    onWarningMessage: Function,
+    onNotificationMessage: Function,
+    onSubscribed: Function,
+    onUnsubscribed: Function,
+    onResending: Function,
+    onResent: Function,
+    onNoResend: Function,
+    onError: Function,
+    subscriptionStatus?: SubscriptionStatus,
+    isActive?: boolean,
+    uiChannel: StreamId,
+    clientContext: ClientContextProps,
+    resendFrom: any,
+    resendTo: any,
+    resendLast: any,
```

Comment:
```
    resendFrom: any,
    resendTo: any,
    resendLast: any,
```
I think these should be of type `number`
https://github.com/streamr-dev/streamr-client-javascript#resending-historical-data